bigquery/setup.py

```diff
@@ -50,7 +50,7 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.21.0',
+    'google-cloud-core >= 0.20.*, < 0.21.0',
```

Comment:
Please follow the pattern we've already established:

```
>= 0.20.0, < 0.21dev'
```

@jonparrott confirm?